src/core/free_for_all_game_mode_system.cpp

```diff
@@ -104,6 +104,7 @@ void FreeForAllGameModeSystem::OnStartGameMode( core::StartGameModeEvent const&
     {
         return;
     }
+#if 0
```

Comment:
do you plan to keep the code or use in the near future? if no, then please don't use #if 0